src/sidebar/components/FilterSelect.tsx

```diff
@@ -1,28 +1,23 @@
 import classnames from 'classnames';
+import type { IconComponent } from '@hypothesis/frontend-shared/lib/types';
```

Comment:
👍🏻 Glad you were able to figure this out.

Reply:
Actually my IDE suggested the import 😅